alot/commands/globals.py

```diff
@@ -1088,6 +1089,28 @@ def apply(self, ui):
             ui.notify('Error when reloading config files:\n {}'.format(e),
                       priority='error')
 
+@registerCommand(MODE, 'theme',     arguments=[
+    (['theme_name'], {'help': 'Name of the theme'})], help='Select your own them')
+class ThemeCommand(Command):
+
+    """move in widget"""
+    def __init__(self, theme_name=None, **kwargs):
```

Comment:
this does not do anything so could be omitted? the docstring is also not correct is it?